packages/@aws-cdk/custom-resources/test/provider-framework/integration-test-fixtures/s3-file-handler/index.ts

```diff
@@ -4,7 +4,7 @@ import api = require('./api');
 
 const s3 = new AWS.S3();
 
-export async function onEvent(event: AWSCDKAsyncCustomResource.OnEventRequest): Promise<AWSCDKAsyncCustomResource.OnEventResponse> {
+export async function onEvent(event: AWSCDKAsyncCustomResource.OnEventRequest) {
```

Comment:
Why strip the explicit return type?

Reply:
because the actual return type is `Promise<AWSCDKAsyncCustomResource.OnEventResponse | void | undefined>` and it's a mouthful.